hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/ContainerStateMachine.java

```diff
@@ -862,4 +864,48 @@ public void notifyLeaderChanged(RaftGroupMemberId groupMemberId,
                                   RaftPeerId raftPeerId) {
     ratisServer.handleLeaderChangedNotification(groupMemberId, raftPeerId);
   }
+
+  @Override
+  public String toStateMachineLogEntryString(StateMachineLogEntryProto proto) {
+    try {
+      ContainerCommandRequestProto requestProto =
+              getContainerCommandRequestProto(proto.getLogData());
+      return getWriteChunkInfo(requestProto);
+    } catch (Throwable t) {
+      return "";
+    }
+  }
+
+  private String getWriteChunkInfo(ContainerCommandRequestProto requestProto) {
+    StringBuilder builder = new StringBuilder();
+    Preconditions.checkArgument(requestProto.getCmdType() == Type.WriteChunk);
```

Comment:
Can we add logic for all the request types or just use a switch case with default option?

Reply:
Done, added a switch case with only one handling for writeChunk.